bower.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "adapt-componentName",
   "description": "DESCRIPTION",
-  "version": "0.0.1",
-  "framework": ">=2.2.0",
+  "version": "1.0.0",
+  "framework": ">=5.8.0",
```

Comment:
Should be v5.14 for jsx fixes https://github.com/adaptlearning/adapt_framework/releases/tag/v5.14.0